<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>882</width>
    <height>672</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="settings_page">
      <widget class="QSplitter" name="splitter">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>310</width>
         <height>252</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="QWidget" name="layoutWidget">
        <layout class="QFormLayout" name="settings_layout">
         <item row="0" column="0">
          <widget class="QLabel" name="von_label">
           <property name="text">
            <string>Von</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="von_combo"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="nach_label">
           <property name="text">
            <string>Nach</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QComboBox" name="nach_combo"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="strecke_label">
           <property name="text">
            <string>Strecke</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QListWidget" name="strecke_list"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="hidden_widget" native="true"/>
      </widget>
     </widget>
     <widget class="QWidget" name="display_page"/>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
